helm/cluster-vsphere/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+appVersion: 1
```

Comment:
We agreed not to use appVersion for cluster-openstack as it doesn't really have a meaning. Happy to revisit, just in the interest of keeping them in sync.